test/evergreen.yml

```diff
@@ -2129,36 +2131,44 @@ tasks:
           unit_test_args: -v 2 --long
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=snappy logging=1 logging_compression=snappy logging_prealloc=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row alter=1 backups=1 compaction=1 data_extend=1 prepare=1 salvage=1 statistics=1 statistics_server=1 verify=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row firstfit=1 internal_key_truncation=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: leak_memory=0 mmap=1 file_type=row checkpoints=0 in_memory=1 reverse=1 truncate=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
           extra_args: checkpoints=1 leak_memory=0 mmap=1 file_type=row compression=zlib huffman_value=1
       - func: "format test"
         vars:
+          config: ../../../test/format/CONFIG.coverage
```

Comment:
I think `leak_memory=0 mmap=1` are common to all runs, so they could go in `CONFIG.coverage`.